# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/remoteworkers/PRemoteWorkerDebugger.ipdl
# Commit: f2debe329bca
# Full Hash: f2debe329bcaaad1248a234c165ccf240d602344
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-04-15 21:21:06
# Regressor Bug: 1899503
# File Overlap Count: 4
# Description:
#   Bug 1899503 - P3 PRemoteWorkerDebugger binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebugger binding.
#   
#   We are trying to keep the same logic debugger registration/unregistration to minimize the difference between the local debugger(the debugger on the content process main thread)
# ==============================================================================

diff -r 604c4c52574a -r f2debe329bca dom/workers/remoteworkers/PRemoteWorkerDebugger.ipdl
--- a/dom/workers/remoteworkers/PRemoteWorkerDebugger.ipdl	Tue Apr 15 09:47:39 2025 +0000
+++ b/dom/workers/remoteworkers/PRemoteWorkerDebugger.ipdl	Tue Apr 15 09:47:39 2025 +0000
@@ -24,6 +24,9 @@
   async RemoveWindowID(uint64_t aWindowID);
 
 child:
+  async RegisterDone();
+  async UnregisterDone();
+
   async Initialize(nsString aURL);
   async PostMessage(nsString aMessage);
   async SetDebuggerReady(bool aReady);